Author: Rasmus Thomsen <oss@cogitri.dev>
Reason: The logic in the configure.ac is wrong: Pandoc is a hard requirement
for the man pages and $srcdir/man/man1 always exists, so we need to use
AND here and not OR.
Upstream: No
diff --git a/configure.ac b/configure.ac
index 7b5c219..4331fd4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -30,7 +30,7 @@ AS_IF(
 AM_CONDITIONAL([HAVE_PANDOC],[test "x${PANDOC}" = "xyes"])
 AM_CONDITIONAL(
     [HAVE_MAN_PAGES],
-    [test -d "${srcdir}/man/man1" -o "x${PANDOC}" = "xyes"])
+    [test -d "${srcdir}/man/man1" -a "x${PANDOC}" = "xyes"])
 
 AC_ARG_ENABLE([fapi],
     [AS_HELP_STRING([--disable-fapi], [disable FAPI tools (default: auto)])],,
